be1-go/hub/election.go

```diff
@@ -157,6 +161,20 @@ func (c *electionChannel) castVoteHelper(publish message.Publish) error {
 		}
 	}
 
+	senderPK := msg.Sender.String()
+	ok = false
+	for _, attendee := range c.attendees {
+		if attendee == senderPK {
+			ok = true
+		}
+	}
```

Comment:
If `attendees` is a map you can simply do `ok := c.attendees[senderPK]`

Reply:
done